validate-markdown/README.md

```diff
@@ -0,0 +1,60 @@
+[![New Relic Experimental header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#new-relic-experimental)
+
+# 🛠️ Validate markdown action
+
+Validates a changelog in markdown format and prints errors if the changelog is invalid.
+
+## Example Usage
+
+Example exiting if errors are found:
+```yaml
+- name: Check if changelog is valid
+  uses: newrelic/release-toolkit/validate-markdown@v1
+```
+
+Example without exiting:
+```yaml
+- name: Check if changelog is valid
+  id: validate
+  uses: newrelic/release-toolkit/validate-markdown@v1
+  with:
+    exit-code: 0
+- run: |
+    if [[ "${{ steps.validate.outputs.valid }}" != "true" ]]; then
+      echo "markdown is not valid" >&2
+      exit 1
+    fi
+```
+
+
+#### Parameters
+
+All parameters are optional and match the ones used for the cli command flag, you can see the values and the defaults in [here](../README.md#validate-markdown))
+
+## Outputs
+
+`valid`: Returns `true` if the changelog is valid
+
+## Contributing
```

Comment:
The _contributing_, _support_ ..., are included only in the "root README" and the one in `validate-markdown` for any particular reason?

Reply:
mmm.. good point, we should add it to all or only to the main readme at least